relayer/src/chain/counterparty.rs

```diff
@@ -451,6 +474,34 @@ pub fn acknowledgements_on_chain(
     Ok((sequences, height))
 }
 
+/// This method expects an [`IdentifiedChannelEnd`] plus a pair of
```

Comment:
Same here. I think what the method does should be specified first, and then mention the types that the method expects.